### ISSUE TITLE
ChromeVox ScreenReader presents some issues

### ISSUE THREAD

**Issue by isaboy** [subjectsplus/SubjectsPlus#1315]
http://www.sandbox.lbcc.linnlibraries.org/SubjectsPlus/subjects/
ChromeVox: https://chrome.google.com/webstore/detail/chromevox/kgejglhpjiefppelpmljglcjbhoiplfn
One big issue is that, since the links aren't properly labelled, the reader reads the links as they literally appear. Not everyone will know what "WR123" is. More often than not, the reader doesn't say that what's being read is a link.
 - According to ChromeVox, each column in a section is its own list, separate of each other. For those who can't see the screen, this can be very problematic.
 - For some portions of the webpages available, lists are read as one runon sentence.
 - Keyboard controls tend to skip very large portions of the webpages.

**Comment by isaboy** [comment]
![image](https://user-images.githubusercontent.com/44935992/48740296-ccd35f00-ec0b-11e8-8a00-b57814db2601.png)
For subjects/themes/um/index_c.php the lines start at 129,
For subjects/themes/med/index_c.php at line 185,
For subjects/index.php at line 196,
For subjects/includes/header_um.php at line 96, etc.

The issue is that the html is creating two lists with one column each, when there should be one list with 2 or more columns. Proper labeling would also help screen readers interpret the names of each entry to the user.

**Comment by sandbergja** [comment]
To fix the issue where a single list is read as multiple lists if it is broken across two columns, we could use CSS3's column functionality to draw those multiple columns instead of using PHP.  It looks like the multi-column layout is pretty widely supported: https://caniuse.com/#feat=multicolumn

I will experiment with this and do a pull request if it works out.

**Comment by sandbergja** [comment]
The CSS-based solution worked well in my (limited) testing with NVDA+Firefox.  Rather than reading the lists as two separate lists, it was able to give the user an accurate count of list items.

For example, if you have four guides, NVDA previously announced "List with two items" twice.  With commit 7c6917c, it announces "List with four items".
